notes-webapp/src/main/webapp/skin/less/notes/notes.less

```diff
@@ -341,6 +291,12 @@
       }
     }
   }
+
+  .content-link {
+    a {
+      font-size: 16px !important;
```

Comment:
We already have CSS helpers to define font sizes of elements, can you use it please instead.
Imagine we want to change the fonts sizes globally, we will have to change in one single place, plus this strategy avoids to have a huge CSS files to parse in browser side (CPU & Memory) and to download (Network performances)

Reply:
which helper that allow to set exactly 16px as fontsize ?

Reply:
`subtitle-1` defines in rem defines the style of subtitles used in product.